src/lib/fetch-file-string.js

```diff
@@ -1,16 +1,6 @@
 async function fetchFileString(url) {
 	const res = await fetch(url)
-	const fileBuffer = await bufferFromStream(res.body)
-	return fileBuffer.toString()
-}
-
-async function bufferFromStream(stream) {
-	return new Promise((resolve, reject) => {
-		const _buf = []
-		stream.on('data', (chunk) => _buf.push(chunk))
-		stream.on('end', () => resolve(Buffer.concat(_buf)))
-		stream.on('error', (err) => reject(err))
-	})
+	return res.text()
 }
```

Comment:
I was getting an error here: `stream.on is not a function`. Shifted to use the `.text()` method on the fetch response instead, as I believe it accomplishes the same result.